.github/workflows/checks.yml

```diff
@@ -97,7 +97,20 @@ jobs:
     steps:
       - name: SCM Checkout
         uses: actions/checkout@v3
+      - name: Free disk space
+        uses: jlumbroso/free-disk-space@main
+        with:
+          tool-cache: true
+          large-packages: false
 
+      - name: Free disk space by removing large directories
+        run: |
+          sudo rm -rf /usr/local/graalvm/
+          sudo rm -rf /usr/local/.ghcup/
+          sudo rm -rf /usr/local/share/powershell
+          sudo rm -rf /usr/local/share/chromium
+          sudo rm -rf /usr/local/lib/node_modules
+          sudo rm -rf /opt/ghc
```

Comment:
Interesting ... WTF is contained in this default images :exploding_head:

Reply:
No idea, I just copy/pasted it from the ai-lab repo. The test were no longer working without it.